go/api/generic_commands.go

```diff
@@ -385,4 +385,23 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: Https://valkey.io/commands/type/
 	Type(key string) (Result[string], error)
+
+	// Removes the existing timeout on key, turning the key from volatile
+	// (a key with an expire set) to persis tent (a key that will never expire as no timeout is associated).
```

Comment:
```suggestion
	// (a key with an expire set) to persistent (a key that will never expire as no timeout is associated).
```

Reply:
fix in commit 81f88c01facad7bd531052e283b75666f2e349cf